cincinnati/src/lib.rs

```diff
@@ -44,7 +42,7 @@ pub enum Release {
 }
 
 impl Release {
-    pub fn version(&self) -> &Version {
+    pub fn version(&self) -> &str {
```

Comment:
Why did you switch from `semver::Version` to `&str` here? Isn't the former much more helpful when comparing versions and such?

Reply:
I just noticed that this was discussed beforehand in https://github.com/openshift/cincinnati/issues/6#issuecomment-433952395.